### ISSUE TITLE
Login sometimes requires manual page reload

### ISSUE THREAD

**Issue by wanLo** [studieren-ohne-grenzen/dashboard-web-frontend#6]
Maybe a hick up with localStorage- & cookie-based authentification?

**Comment by laurenzfg** [comment]
Yes i had this issue. I guess the problem is upstream. After all, we use the upstream version of nuxt-auth-next

**Comment by wanLo** [comment]
It actually was already there with nuxt-auth v4. But still might be an issue we can't solve ourselves?

**Comment by laurenzfg** [comment]
I've seen it once in the console. It occurs if the hybrid online-offline thingy is in a specific state and thus a cookie cannot be set. I don't know what we can do agains it (although I'm not an expert on Nuxt at all).
